ihm/multi_state_scheme.py

```diff
@@ -0,0 +1,225 @@
+# coding=utf-8
+
+"""Classes for handling connected/ordered schemes formed by multiple state
+    together with information on kinetic schemes"""
+import ihm.multi_state_scheme
+
+
+class MultiStateScheme(object):
+    """MultiStateScheme collects information about a collection of
+       multiple states, that can form a connected/ordered scheme.
+       A special case is a kinetic scheme, for which kinetic rates and
+       relaxation times are available.
+
+       :param str name: The name of the multi-state scheme.
+       :param str details: Details on the scheme.
+       :param list_of_connectivities: A list of connectivities that belong to
+        the scheme.
+       :type list_of_connectivities: List of
+        :class:`ìhm.MultiStateSchemeConnectivity`
+       :param list_of_relaxation_times: A list of relaxation times not assigned
+        to specific connectivities, but to the scheme
+       :type list_of_relaxation_times: List of :class:`ihm.RelaxationTime`
+    """
+    def __init__(self, name, details=None, list_of_connectivities=None,
+                 list_of_relaxation_times=None):
+        self.name = name
+        self.details = details
+        self.connectivity_list = []
+        self.relaxation_time_list = []
+        self.states = []  # filled automatically based on the connectivity_list
```

Comment:
Consider making these last three members private, e.g. `self._connectivity_list`. That way they will not be part of the API and so can be easily changed in future if needed.